encoding/_toml/parser_test.ts

```diff
@@ -0,0 +1,441 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+import {
+  assertEquals,
+  assertThrows,
+} from "https://deno.land/std@0.101.0/testing/asserts.ts";
```

Comment:
This also needs to be relative path dependency.

Reply:
And fixed too!